@xen-orchestra/web/src/components/site/dashboard/BackupIssues.vue

```diff
@@ -27,11 +27,13 @@ import { useI18n } from 'vue-i18n'
 
 const { t } = useI18n()
 
-const { backupIssues, isReady } = useDashboardStore().subscribe()
+const { record } = useDashboardStore().subscribe()
 
-const hasBackupIssues = computed(() => backupIssues.value.length !== 0)
+const backupsIsReady = computed(() => record.value?.backups?.issues !== undefined)
 
-const coreBackupIssues = computed(() => backupIssues.value.map(convertBackupIssueToCore))
+const hasBackupIssues = computed(() => backupsIsReady.value && record.value!.backups!.issues.length > 0)
```

Comment:
```suggestion
const hasBackupIssues = computed(() => (record.value?.backups?.issues.length ?? 0) > 0)
```
I think it's less prone to potential errors, what do you think about that ?